mobile/src/src/components/Housing/Listings/ListingView.jsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { StyleSheet, Dimensions, Image, View, Text } from "react-native";
+import { Card, Title, Paragraph, Avatar } from 'react-native-paper';
+import ProfilePhoto from "../../common/ProfilePhoto";
+import { Choices } from "../../../utils/endpoints";
+
+const ListingView = (props) => {
+
+    const {width, height} = Dimensions.get('window');
+
+    return (
+        <Card style={styles.container}>
+            <Card.Content>
+                <View style={{ flexDirection: "row" }}>
+                    <View style={{flex: 1}}>
+                        <ProfilePhoto photo={props.creator.photo}></ProfilePhoto>
+                    </View>
+                    <View style={{flex: 5, flexDirection: "column"}}>
+                        <Text>{props.creator.preferred_name}</Text>
+                        <Text style={{color: "grey"}}>{props.datetime_created.substring(0, "YYYY-MM-DD".length)}</Text>
+                    </View>
+                </View>
+                <Title style={styles.title}>{props.title}</Title>
+                <Paragraph style={styles.bodyText}>{props.content}</Paragraph>
+                <Paragraph style={styles.bodyText}>{"Region: " + Choices.regions[props.region]}</Paragraph>
+                <Paragraph style={styles.bodyText}>{"Price: $" + props.price}</Paragraph>
+                <Paragraph style={styles.bodyText}>{"Term: " + Choices.seasons[props.season] + " " + props.year}</Paragraph>
+                <Paragraph style={styles.bodyText}>{"Category: " + Choices.listingCategories[props.category]}</Paragraph>
+                <Image source={{uri: props.photo}}
```

Comment:
@trentstauff @shashankanandkannan if the photo is null, we don't need to put `<Image style={{ width: 0, height: 0 }} />`. We can simply do:
`{photo && <Image source={{ uri: photo }} style={{ width: 0.88 * width, height: 300 }} /> }`
What this does is, if photo is true (is not null and not false) then it returns the Image. Otherwise, it returns nothing.

Reply:
I agree